ext/intl/breakiterator/breakiterator.stub.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+class IntlBreakIterator implements Traversable
+{
+    /** @return IntlBreakIterator|null */
+    public static function createCharacterInstance(?string $locale = null) {}
+
+    /** @return IntlCodePointBreakIterator */
+    public static function createCodePointInstance() {}
+
+    /** @return IntlBreakIterator|null */
+    public static function createLineInstance(?string $locale = null) {}
+
+    /** @return IntlBreakIterator|null */
+    public static function createSentenceInstance(?string $locale = null) {}
+
+    /** @return IntlBreakIterator|null */
+    public static function createTitleInstance(?string $locale = null) {}
+
+    /** @return IntlBreakIterator|null */
+    public static function createWordInstance(?string $locale = null) {}
+
+    private function __construct() {}
+
+    /** @return int */
+    public function current() {}
+
+    /** @return int */
+    public function first() {}
+
+    /** @return int|false */
+    public function following(int $offset) {}
+
+    /** @return int|false */
+    public function getErrorCode() {}
```

Comment:
`false` killed in https://github.com/php/php-src/commit/16897aacce85df66a34a04d3b5a84bbe7b8c4871.